metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
+    article = models.ForeignKey(Article)
+    topic = models.ForeignKey(Topic)
+    owner = models.ForeignKey(User)
+
+    location = models.IntegerField()
```

Comment:
An index in the article, sure. But then we definitely need to store some kind of container ID. And if we're smart and compute some kind of hash of the article to detect changes, we can inform users that things have changed (and even where they have changed if we use some differencing library).